src/xcb_ffi/raw_ffi.rs

```diff
@@ -129,57 +132,182 @@ pub(crate) mod send_request_flags {
     pub(crate) const REPLY_FDS: c_int = 8;
 }
 
+#[cfg(all(not(test), feature = "dl-libxcb"))]
+struct LibxcbLibrary {
+    // Needed to keep the library loaded
+    _library: libloading::Library,
+    funcs: LibxcbFuncs,
+}
+
+#[cfg(all(not(test), feature = "dl-libxcb"))]
+impl LibxcbLibrary {
+    fn open_lib() -> Result<libloading::Library, LibxcbLoadError> {
+        // TODO: Names for non-unix platforms
+        #[cfg(unix)]
+        const LIB_NAMES: &[&str] = &["libxcb.so.1", "libxcb.so"];
```

Comment:
Why both the full SONAME and also the short version? Shouldn't either be enough?

(I'd say `libxcb.so.1` is the way to go, but I am not entirely sure...)

Reply:
Indeed, if we ever have a "libxcb.so.2" and a "libxcb.so" that links to it, we should not try to use it because there is presumably ABI/API breakage, so I'll change it to a single name.